src/MainView.vala

```diff
@@ -95,64 +101,84 @@ public class DateTime.MainView : Gtk.Grid {
         };
         time_zone_picker.get_style_context ().add_class (Gtk.STYLE_CLASS_FRAME);
 
-        var show_date_label = new Gtk.Label (_("Show the date:")) {
-            halign = Gtk.Align.END,
-            margin_top = 24
+        var week_number_label = new Gtk.Label (_("Show week numbers:")) {
+            halign = Gtk.Align.END
         };
 
-        var show_date_switch = new Gtk.Switch () {
+        var week_number_switch = new Gtk.Switch () {
             halign = Gtk.Align.START,
-            valign = Gtk.Align.CENTER,
-            margin_top = 24
+            valign = Gtk.Align.CENTER
+        };
+
+        var week_number_info = new Gtk.Label (_("e.g. in the Calendar app and Panel indicator")) {
```

Comment:
Nice job with the hint text :+1:

Reply:
Isn't "e.g." too technical? I'd use "for example". Just my 2¢.

Reply:
@lainsce It doesn't read as technical to me and I think it could go either way, but `e.g.` was much more concise. :)

Reply:
I'm just thinking of the eventual i18n work hehe